_data/toc.yaml

```diff
@@ -1280,9 +1280,27 @@ manuals:
               path: /desktop/extensions-sdk/tutorials/minimal-frontend-using-docker-cli/
             - title: Create a minimal backend extension
               path: /desktop/extensions-sdk/tutorials/minimal-backend-extension/
-        - title: Design Guidelines
-          path: /desktop/extensions-sdk/design/design-overview/
-        - sectiontitle: Extensions
+        - sectiontitle: Design
+          section:
+             - title: Guidelines
+               path: /desktop/extensions-sdk/design/design-guidelines/
+             - title: Docker's design principles
+               path: /desktop/extensions-sdk/design/design-principles/
+        - sectiontitle: UI Styling
+          section:
+             - title: Overview
+               path: /desktop/extensions-sdk/design/overview/
+             - title: MUI best practices
+               path: /desktop/extensions-sdk/design/mui-best-practices/
+        - sectiontitle: Developer SDK tools
+          section:
+            - title: Overview
+              path: /desktop/extensions-sdk/dev/overview/
```

Comment:
Is it possible to use index pages for each level in the URL tree, so that navigating the URL structure leads to useful content?

I.e.;

- `/desktop/extensions-sdk/` -> overview of the extensions SDK (even wondering if this should be `/desktop/extensions/sdk/`)
- `/desktop/extensions-sdk/design/` -> overview of design
- `/desktop/extensions-sdk/dev/` -> overview of SDK tools
- etc.

For example, currently

- https://docs.docker.com/desktop/extensions-sdk/tutorials/initialize/ Is the overview of "Extensions SDK tutorials"
- but navigating one level up (which would be a natural thing to expect to go "a level up" is a 404; https://docs.docker.com/desktop/extensions-sdk/tutorials/

Reply:
Yes, so renaming (for example); https://github.com/docker/extensions-sdk/blob/main/docs/dev/overview.md to `index.md`, that way it automatically becomes the index of that directory, and the URL would become `/desktop/extensions-sdk/dev/`

Reply:
We can do that separately (and add a redirect for the old location), but it'd be good to have the "landing" page for each directory to be named `index.md`